### PR TITLE
Front 2.0

### PR DESCRIPTION
**Pull request by patte**
besids other stuff:
- [x] merge all translations into one file

After merge, delete these gdocs:
- https://docs.google.com/spreadsheets/d/1NE1igcbxtTrVd5P72Ubqmgv4yKYh0LRGa0EuYSEQKxE/edit#gid=0
- https://docs.google.com/spreadsheets/d/1PckJT0ft4snNWmtlJHHXN93-Rk8eyalHsDCjfA1wc5g/edit#gid=0
- https://docs.google.com/spreadsheets/d/1_1g3sW8u1aD9xGRfS479s0Pjwx0LxYRUEhETGkKlwLE/edit#gid=0
- https://docs.google.com/spreadsheets/d/1mgLz3k2rQ_qcB2ZpmJRZbGLAX5vGcr0P9I2jbWaIgog/edit#gid=0

new file: https://docs.google.com/spreadsheets/d/1ZzaC_F7KKP5nSP36n6D9Q6wyTHeP037TBg4kF41jgS4/edit#gid=0

### CODE REVIEW

**Comment by tpreusse** [comment]
Frontend-Integration: ok :)

File: packages/discussions/graphql/resolvers/_queries/comments.js [hunk 1]
```diff
@@ -83,13 +83,16 @@ module.exports = async (_, args, context, info) => {
...
+    id: `${discussionId || 'comments'}${offset || ''}`,
     totalCount: numComments,
     directTotalCount: numComments,
     pageInfo: {
       hasNextPage: !!endCursor,
       endCursor
     },
-    nodes: comments
+    nodes: comments,
+    focus: focusId && focusId === comments[0].id
```
Comment (patrickvenetz) [inline packages/discussions/graphql/resolvers/_queries/comments.js]: Safety question: Can `comments` be an empty array and if so will this cause and error?

File: packages/auth/lib/t.js [hunk 2]
```diff
@@ -1,4 +1,2 @@
-const { getFormatter } = require('@orbiting/backend-modules-translate')
-const MESSAGES = require('./translations.json').data
-
-module.exports = getFormatter(MESSAGES)
+const { t } = require('@orbiting/backend-modules-translate')
+module.exports = t
```
[the remark this answers is not in the document]
Comment (patte) [reply packages/auth/lib/t.js]: Not completely lost it's purpose, it's the connector for the auth package and would make changing `t` in auth simpler.

File: packages/discussions/graphql/resolvers/_queries/comments.js (hunk 1 above)
Comment (patte) [reply packages/discussions/graphql/resolvers/_queries/comments.js]: good catch, fixed!

**Pull request merged**